apic_ml2/neutron/services/l3_router/l3_apic.py

```diff
@@ -266,50 +120,24 @@ def remove_router_interface(self, context, router_id, interface_info):
     def create_floatingip(self, context, floatingip):
         res = super(ApicL3ServicePlugin, self).create_floatingip(
             context, floatingip)
-        port_id = floatingip.get('floatingip', {}).get('port_id')
-        self._notify_port_update(port_id)
-        if res:
-            res['status'] = self._update_floatingip_status(context, res['id'])
+        fip_context = api.FipContext(res, original=floatingip)
```

Comment:
The use of 'current' and 'original' is strange and quite different from ML2/GBP. 'original' is set only for the "update" operation, and is the DB state before the update operation. 'current' is the new state of the object for create and update, or the existing DB state for delete.